src/services/submission_processor.py

```diff
@@ -26,6 +26,12 @@ async def upload_to_storage(lei: str, submission_id: str, content: bytes, extens
         raise HTTPException(status_code=HTTPStatus.INTERNAL_SERVER_ERROR, detail="Failed to upload file")
 
 
+async def upload_to_storage_and_update_submission(lei: str, submission_id: str, content: bytes, extension: str = "csv"):
+    await upload_to_storage(lei, submission_id, content, extension)
+    # Set SUBMISSION_UPLOADED
+    await update_submission(SubmissionDAO(submitter=submission_id, state=SubmissionState.SUBMISSION_UPLOADED))
+
+
```

Comment:
This can be removed.  Setting SUBMISSION_UPLOADED is part of the upload story I'm doing.  This would be an incorrect way to update a submission, also.  But once my story is in, it'll make more sense.  So just ignore SUBMISSION_UPLOAD in here.